docs/docs/learn/introduction/gateway/index.md

```diff
@@ -0,0 +1,64 @@
+1. The user sends BTC to the liquidity provider's Bitcoin address. A hash of the user's order is included in the `OP_RETURN` of their transaction, including data such as the recipient's EVM address on BOB and their desired strategy (i.e. their intent).
+1. The relayer submits a Merkle proof of the user's Bitcoin transaction to an on-chain [Light Client](/learn/builder-guides/relay), granting the relayer permission to withdraw the LP's wrapped Bitcoin. **Encoding the user's order in their Bitcoin transaction makes it possible to trustlessly verify and complete the user's intent without using an oracle.**
+1. Gateway sends the LP's wrapped Bitcoin to the user's EVM address. If the user requested a Bitcoin LST/LRT, that token is minted using the LP's wrapped Bitcoin _before_ it is sent to the user.
+
+## Architecture
+
+<img
+src={require("./architecture.png").default}
+style={{ width: "100%", maxWidth: "100%", height: "auto" }}
+alt="architecture"
+/>
+
+### User Flow
+
+1. A user requests to swap BTC for wrapped BTC (e.g. WBTC, tBTC, or FBTC) or staked BTC (e.g. SolvBTC.BBN, uniBTC).
+1. The user gets a "quote" of which Gateway smart contract is an available route (i.e. which LP they can swap with).
+1. The user creates an "order" with the relayer to reserve the LP's liquidity.
+1. The user creates a Bitcoin transaction, updating the order with their txid. Gateway's SDK creates a hash of the user's intent, which is included in the `OP_RETURN` of the transaction. This hash includes metadata such as the user's EVM address, which strategy they are using (i.e. their intent), and how many sats to convert to ETH for gas, among other data. By including a deterministic hash of the user's order, we lay the groundwork for making Gateway trustless in the future by decentralizing the relayer set.
+1. The relayer monitors the Bitcoin chain and sends the LP's wrapped BTC to the user after the txid is seen on Bitcoin. Specifically, the relayer submits a Merkle proof of the user's Bitcoin tx to an [on-chain light client](/learn/builder-guides/relay) for trustless verification of the user's intent. It accomplishes this by requiring that the recipient's EVM address, sequence of smart contract calls, and other order data exactly match the hash in the `OP_RETURN` of the user's Bitcoin transaction.
+
+### Liquidity Provider (LP) Flow
+
+1. An LP asks the relayer to deploy a new Gateway contract, which functions as an escrow for their funds. This is permissioned at the moment because BOB pays the transaction's gas. See the [Trust Assumptions](#trust-assumptions) section below for more information.
+2. The LP deposits wrapped Bitcoin (e.g. WBTC, tBTC, FBTC) in their Gateway contract.
+3. The LP can only withdraw their funds or update their swap fees after a delay so that the relayer has time to finish open orders. The relayer will not accept new orders during this delay until reset.
+
+## Trust Assumptions
```

Comment:
@gregdhill could you please double check if this is accurate?

Reply:
`setFeeRanges` has the `canUpdate` modifer, which references `UPDATE_DELAY = 6 hours`. I checked a sample [live V3 contract](https://explorer.gobob.xyz/address/0xcc2c1d45A3ECeEdec364a6B9e537AE97Fa20bEa7?tab=contract_code) to verify that all was included in prod.
[Gateway.sol source](https://github.com/bob-collective/bob-gateway/blob/master/contracts/src/Gateway.sol)